CMakeLists.txt

```diff
@@ -142,6 +143,7 @@ function(openvino_developer_export_targets)
         "A list of OpenVINO exported components" FORCE)
 endfunction()
 
+add_subdirectory(thirdparty)
```

Comment:
Please add thirdparty directory after the openvino.
Because in the thirdparty we have an itt collector which depends on ittnotify.

Reply:
maybe we need to put `ittnotify` to `thirdparty` as well?

Logically, `thirdparty` should come first..

Reply:
Hmmm, ok, I created a PR: https://github.com/openvinotoolkit/openvino/pull/4164